### PR TITLE
poetry: remove list command

### PR DESCRIPTION
**Pull request by yenaras**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
1.1.14

### CODE REVIEW

File: pages/common/poetry.md [hunk 1]
```diff
@@ -30,7 +30,3 @@
 - Bump the minor version of the project in `pyproject.toml`:
 
 `poetry version minor`
-
-- List all poetry subcommands:
-
-`poetry list`
```
Comment (mfrw) [inline pages/common/poetry.md:36]: The documentation does say there exists a `poetry list` subcommand.
Reference: https://python-poetry.org/docs/cli/#list
Not sure if the docs are uptodate.

Comment (yenaras) [reply pages/common/poetry.md:36]: When I run it locally it says `The command "list" is not defined.`

image

**Review by kbdharun — APPROVED** [review]

File: pages/common/poetry.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/poetry.md:36]: > When I run it locally it says `The command "list" is not defined.`
> 
> image

Yeah, this is known, and the command was removed upstream in favour of `poetry` alone to list all commands.
Ref: https://github.com/python-poetry/poetry/issues/1798, https://github.com/python-poetry/poetry/pull/1800, https://github.com/python-poetry/poetry/tree/master/src/poetry/console/commands

Comment (yenaras) [reply pages/common/poetry.md:36]: Thank you, I will go to poetry github and submit a pr to remove it from their docs if I can. <3

Comment (yenaras) [reply pages/common/poetry.md:36]: It seems someone already sent a PR for it actually, glad to have helped with tldr though :)

**Review by mfrw — APPROVED** [review]
LGTM
Thank you @yenaras

**Pull request merged**